src/components/Clock/Clock.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './Clock.scss';
+
+class Clock extends React.Component {
+  state = {
+    time: new Date().toLocaleTimeString,
+  }
+
+  componentDidMount() {
+    this.interval = setInterval(() => {
+      this.setState({ time: new Date().toLocaleTimeString() });
+      // eslint-disable-next-line no-console
+      console.log(this.state.time);
+    }, 1000);
+  }
+
+  componentDidUpdate(props) {
```

Comment:
```suggestion
  componentDidUpdate(prevProps) {
```
For better understanding